app/assets/stylesheets/finder_frontend.scss

```diff
@@ -133,17 +89,16 @@
     }
   }
 
-  .filter-form,
-  .filtered-results {
-    @include media(tablet) {
-      float: left;
-      width: $one-third;
-    }
+  .filtering {
+    clear: both; // Clear `header` float for max-width
```

Comment:
now that `header` is a grid row I don't this this is needed. Grid rows include our containing floats shim.